.gitignore

```diff
@@ -0,0 +1,4 @@
+# IDE
+.idea
```

Comment:
I have no expectations on ignoring any files inside the Foundation repository any time soon, so adding `.gitignore` just for the sake of dealing with IDE-related stuff seems a little bit excess from my perspective.

You can instead configure your local git to globally ignore IDE folders in any repository by doing the following:

```bash
git config --global core.excludesfile ~/.gitignore
echo ".idea" >> ~/.gitignore
```

Reply:
Deal, remove gitignore